### PR TITLE
Update firefox/mobile/ english pages with new brand (#6987)

### LINKED ISSUE #6987: Update /firefox/mobile/ with new branding
## Description

Update the Firefox Mobile landing page at https://www.mozilla.org/en-US/firefox/mobile/ with a temporary English only page that reflects the new brand.

***Tasks***
- [ ] Make the page available for stakeholder review on demo server
- [ ] Ensure UTMs persist from the existing page so we can measure historically
- [ ] Have @rraue review the UTMs

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Add a temporary English only firefox/mobile page with new branding.
- Protocol style sheet for send to device widget
- Protocol style sheet for firefox subnav

## Issue / Bugzilla link

Fix #6987

## Testing
- [x] shows to all English locales
- [x] send to device widget (opens when clicking blue "Get Firefox" button)
- [x] Focus QR code
- [ ] good fallbacks for old browsers
- [x] Google Play or Apple App store images appear if user is already in Android or iOS

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Animations are not working in IE or Edge 😕
Integration tests are failing.

**Comment by stephaniehobson** [comment]
Updates based on design review:

- [ ] narrow boxes in privacy section
- [ ] reduce spacing between heading and text in privacy section
- [ ] reduce max-width of text area for accounts section
- [ ] line beneath android only items to max 20 px to left and right of phone
- [ ] bottom call out headline 48px font
- [ ] more padding above focus logo

File: bedrock/firefox/urls.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
                                       template_context_variations=['a']),
         name='firefox.features.private-browsing'),
     url(r'^firefox/ios/testflight/$', views.ios_testflight, name='firefox.ios.testflight'),
-    page('firefox/mobile', 'firefox/mobile.html'),
+    url(r'^firefox/mobile', views.firefox_mobile, name='firefox.mobile'),
```
Comment (alexgibson) [inline bedrock/firefox/urls.py]: I took a quick peek and the failing integration tests seem to be redirects. I think changing this to `r'^firefox/mobile/$'` should fix the issue.

Comment (stephaniehobson) [reply bedrock/firefox/urls.py]: Thanks @alexgibson!

**Comment by stephaniehobson** [comment]
DNM: Hold this page to put it live the same time we put the updated desktop page live.

File: bedrock/firefox/templates/firefox/mobile-en.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+          <div class="header-product-ctas">
+            <button id="get-focus" type="button" class="js-focus mzp-c-button mzp-t-secondary mzp-t-dark" data-product="focus" data-button-name="Send to SMS Modal - Firefox Focus">{{ _('Get Firefox Focus') }}</button>
+          </div>
+          <div class="mobile-download-buttons-wrapper">
+            <ul class="mobile-download-buttons focus" id="mobile-download-buttons-focus">
+              <li class="android">
+                {{ google_play_button(href=settings.GOOGLE_PLAY_FIREFOX_FOCUS_LINK, id='playStoreLink', product='Focus') }}
+              </li>
+              {% if l10n_has_tag('focus_apk_link_012018') %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/mobile-en.html]: No need for this l10n tag?

File: media/css/firefox/mobile-protocol.scss [hunk 3]
```diff
@@ -0,0 +1,639 @@
...
+    .t-extend,
+    .t-theme {
+        img {
+            animation: none;
+        }
+    }
+}
+
+@keyframes custom {
+  0% {
```
Comment (alexgibson) [inline media/css/firefox/mobile-protocol.scss]: Nit: indentation

File: media/img/firefox/mobile/protocol/extend-eye.svg [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="97px" height="90px" viewBox="0 0 97 90" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 53.2 (72643) - https://sketchapp.com -->
```
Comment (alexgibson) [inline media/img/firefox/mobile/protocol/extend-eye.svg]: Would be nice to optimize these svgs?

File: tests/functional/firefox/test_mobile.py [hunk 5]
```diff
@@ -8,64 +8,122 @@
...
 def test_get_firefox_send_to_device_fails_when_missing_required_fields(base_url, selenium):
     page = FirefoxMobilePage(selenium, base_url).open()
-    modal = page.click_get_firefox_header_button()
+    modal = page.click_get_firefox_button()
+    assert modal.is_displayed
+    with pytest.raises(TimeoutException):
+        page.send_to_device.click_send()
+
+
+# Sandstone
```
Comment (alexgibson) [inline tests/functional/firefox/test_mobile.py]: `# Pebbles`?

**Comment by alexgibson** [comment]
Modal content could do with a little less padding on small viewports?

Screenshot_2019-04-09 Get the mobile browser built for you, not advertisers Firefox

**Comment by alexgibson** [comment]
Main heading feels a little large at medium viewport sizes?

**Comment by alexgibson** [comment]
Not sure if intentional, but there's quite a bit more spacing above the mobile download buttons in the main CTA, compared to the regular download button.

Screenshot_2019-04-09 Get the mobile browser built for you, not advertisers Firefox

**Comment by alexgibson** [comment]
On IE9, the hero background picture overlaps the hero copy:

Screenshot 2019-04-09 at 10 37 43

File: bedrock/firefox/templates/firefox/mobile-en.html [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <section class="c-private t-private mzp-t-firefox">
+      <div class="mzp-l-content">
+        <h2>{{ _('Stay invisible on the go with Private Browsing mode') }}</h2>
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/mobile-en.html]: This section talks only about Private Browsing mode, but Tracking Protection is now enabled by default on Firefox for iOS, which is significantly more private than our messaging here suggests? I'm not sure if this is the case for Android yet as I don't own a device, but worth double checking and/or calling out?

File: tests/functional/firefox/test_mobile.py (hunk 5 above)
Comment (stephaniehobson) [reply tests/functional/firefox/test_mobile.py]: ✅

File: media/img/firefox/mobile/protocol/extend-eye.svg (hunk 4 above)
Comment (stephaniehobson) [reply media/img/firefox/mobile/protocol/extend-eye.svg]: ✅

File: bedrock/firefox/templates/firefox/mobile-en.html (hunk 6 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/mobile-en.html]: Hmmmm. We were not trying to update the page content in this iteration. I think this is probably an important update to make but I don't want to block on this.

File: bedrock/firefox/templates/firefox/mobile-en.html (hunk 2 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/mobile-en.html]: ✅

File: media/css/firefox/mobile-protocol.scss (hunk 3 above)
Comment (stephaniehobson) [reply media/css/firefox/mobile-protocol.scss]: ✅

**Comment by stephaniehobson** [comment]
Updated! 

IE9 fix is untested. I will test tomorrow if you don't get to it before then.

**Comment by alexgibson** [comment]
@stephaniehobson think you may still need to push the updated commit here, I'm not seeing any changes?

**Comment by stephaniehobson** [comment]
🤦‍♀️ Pushed.

**Comment by stephaniehobson** [comment]
Looks okay in IE9 to me. Tested on Window IE 11 simulating 9.

**Comment by alexgibson** [comment]
> Looks okay in IE9 to me. Tested on Window IE 11 simulating 9.

All other changes look good, although the IE9 fix doesn't quite seem to work. Inspecting the styles it applies a value of `background-position-x: 50vw` even though it doesn't understand the value :(

Screenshot 2019-04-11 at 09 05 19

**Comment by stephaniehobson** [comment]
Decreasing the priority of this since launch has been delayed by the results of the brand a/b test on /firefox.

**Comment by stephaniehobson** [comment]
We're going to run more tests on /firefox before launching this. I'll revive it when we're ready.

**Pull request closed without merging**